src/main/java/pl/edu/zut/mad/schedule/ScheduleController.java

```diff
@@ -20,17 +21,19 @@
 
     private final GroupAlbumRepository groupAlbumRepository;
     private final ScheduleRepository scheduleRepository;
+    private final ScheduleMapper scheduleMapper;
 
     @Autowired
     public ScheduleController(GroupAlbumRepository groupAlbumRepository,
-                              ScheduleRepository scheduleRepository) {
+                              ScheduleRepository scheduleRepository, ScheduleMapper scheduleMapper) {
```

Comment:
I think it would like better when you move `ScheduleRepository` to previous line